components/director/internal/operations_manager/operations_manager.go

```diff
@@ -176,7 +177,7 @@ func (om *OperationsManager) StartRescheduleOperationsJob(ctx context.Context) e
 			defer om.transact.RollbackUnlessCommitted(ctx, tx)
 			ctx = persistence.SaveToContext(ctx, tx)
 
-			if err := om.opSvc.RescheduleOperations(ctx, om.opType, om.cfg.OperationReschedulePeriod); err != nil {
+			if err := om.opSvc.RescheduleOperations(ctx, om.opType, om.cfg.OperationReschedulePeriod, operationStatuses); err != nil {
```

Comment:
do not forget to adjust the `om.cfg.OperationReschedulePeriod` value to a time which is desired only for system field discovery engine